docs/source/advisor.rst

```diff
@@ -0,0 +1,28 @@
+=========================
+Scylla Monitoring Advisor
+=========================
+
+The Advisor is both a concept and an implementation.
```

Comment:
This is just confusing.
the state of Israel is also concept and an implementation

I suggest something like:

Scylla Advisor is an element of Scylla Monitoring which recognize bad practice usage and advice on how to fix it